lib/game.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../lib/board.rb'
+class Game
+  attr_accessor :board_tic, :player1, :player2, :winning_moves
+
+  def initialize(player1, player2)
+    @board_tic = Board.new
+    @player1 = player1
+    @player2 = player2
+    @winning_moves = [
+      [1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7],
+      [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]
+    ]
+  end
+
+  def mark_board(player)
+    return if @board_tic.marked?(player)
+
+    @board_tic.game_board[0][player.next_move - 1] = player.token if player.next_move.between?(1, 3)
+    @board_tic.game_board[1][player.next_move - 4] = player.token if player.next_move.between?(4, 6)
+    @board_tic.game_board[2][player.next_move - 7] = player.token if player.next_move.between?(7, 9)
+    @board_tic.slots_taken << player.next_move && player.moves << player.next_move
+  end
+
+  def winner?(player)
+    winning_moves.any?(player.moves)
+  end
```

Comment:
If you store the board state into an array, you can use the `all_equal` method to check if there is a winner.
It doesn't work on some horizontal wins.
![Screenshot from 2020-08-26 19-32-43](https://user-images.githubusercontent.com/51926937/91370080-02d74700-e7d3-11ea-9b7b-65b6a1d339d1.png)